crates/cheatnet/tests/cheatcodes/generate_random_felt.rs

```diff
@@ -0,0 +1,56 @@
+use cairo_vm::Felt252;
+use num_bigint::BigUint;
+use num_traits::One;
+
+use cheatnet::runtime_extensions::forge_runtime_extension::cheatcodes::generate_random_felt::generate_random_felt;
+
+#[test]
+fn test_generate_random_felt_range() {
+    let max_felt252: Felt252 = Felt252::from(BigUint::one() << 252);
+
+    assert!(
+        generate_random_felt() < max_felt252,
+        "Number is out of felt252 range"
+    );
+    assert!(
+        generate_random_felt() < max_felt252,
+        "Number is out of felt252 range"
+    );
+    assert!(
+        generate_random_felt() < max_felt252,
+        "Number is out of felt252 range"
+    );
+    assert!(
+        generate_random_felt() < max_felt252,
+        "Number is out of felt252 range"
+    );
+    assert!(
+        generate_random_felt() < max_felt252,
+        "Number is out of felt252 range"
+    );
+}
```

Comment:
I don't think this is a reliable way to test a random number generator. Don't think this test is introducing much.

We could refactor `generate_random_felt` so rng is injected to it but not sure if this is necessary in this case.